src/FakeItEasy/MethodInfoExtensions.cs

```diff
@@ -171,9 +171,14 @@ private static void AppendParameters(StringBuilder builder, ParameterInfo[] para
             {
                 AppendParameterSeparator(builder, i);
                 builder
-                    .Append(parameters[i].ParameterType)
-                    .Append(' ')
-                    .Append(parameters[i].Name);
+                    .Append(parameters[i].ParameterType);

```

Comment:
nit: Maybe drop the line break here? It made sense when there were multiple calls to Append, but now it looks a bit strange.

Reply:
done!